src/main/java/uk/gov/hmcts/juror/job/execution/jobs/dashboard/ams/AmsDashboardGenerateJob.java

```diff
@@ -41,7 +36,7 @@ public AmsDashboardGenerateJob(SchedulerServiceClient schedulerServiceClient,
     @Override
     public List<ResultSupplier> getResultSuppliers() {
         DashboardData dashboardData =
-            new DashboardData(schedulerServiceClient, databaseService, smtpService, config, clock);
+            new DashboardData(schedulerServiceClient, databaseService, null, config, clock);
```

Comment:
It will be work removing the smptService from Dashboard data instead of nulling it here if it is no longer used

Reply:
Updated